Formula/pulumi.rb

```diff
@@ -34,17 +35,10 @@ def install
       bin.install Dir["#{buildpath}/bin/*"]
       prefix.install_metafiles
 
-      # Install bash completion
-      output = Utils.popen_read("#{bin}/pulumi gen-completion bash")
-      (bash_completion/"pulumi").write output
-
-      # Install zsh completion
-      output = Utils.popen_read("#{bin}/pulumi gen-completion zsh")
-      (zsh_completion/"_pulumi").write output
-
-      # Install fish completion
-      output = Utils.popen_read("#{bin}/pulumi gen-completion fish")
-      (fish_completion/"_pulumi").write output
```

Comment:
Yeah, I think this just needs to change to `(fish_completion/"_pulumi.fish").write output`?

Reply:
yeah, either way works (also bash completion is wrong, fixed in the PR)

Reply:
Cool. Using the existing (previous to this PR) methods sounds good to me to preserve existing convention, then.